aspnetcore/fundamentals/middleware/index.md

```diff
@@ -191,7 +191,8 @@ ASP.NET Core ships with the following middleware components, as well as a descri
 | [Authentication](xref:security/authentication/identity) | Provides authentication support. | Before `HttpContext.User` is needed. Terminal for OAuth callbacks. |
 | [CORS](xref:security/cors) | Configures Cross-Origin Resource Sharing. | Before components that use CORS. |
 | [Diagnostics](xref:fundamentals/error-handling) | Configures diagnostics. | Before components that generate errors. |
-| [Forwarded Headers/HTTP Overrides](/dotnet/api/microsoft.aspnetcore.builder.forwardedheadersextensions) | Forwards proxied headers onto the current request. | Before components that consume the updated fields (examples: Scheme, Host, ClientIP, Method). |
+| [Forwarded Headers](/dotnet/api/microsoft.aspnetcore.builder.forwardedheadersextensions) | Forwards proxied headers onto the current request. | Before components that consume the updated fields (examples: Scheme, Host, ClientIP, Method). |
+| [HTTP Method Override](/dotnet/api/microsoft.aspnetcore.builder.httpmethodoverrideextensions) | Allows incoming POST request to override method type. | Before components that consume the updated fields (examples: Scheme, Host, ClientIP, Method). |
```

Comment:
Only the Method field is updated.

Reply:
I'll update to "method field," but note the API comments for the overloads: https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.builder.httpmethodoverrideextensions